packages/SystemUI/src/com/android/systemui/biometrics/FODCircleViewImpl.java

```diff
@@ -46,7 +46,7 @@ public FODCircleViewImpl(Context context, CommandQueue commandQueue) {
     public void start() {
         PackageManager packageManager = mContext.getPackageManager();
         if (!packageManager.hasSystemFeature(PackageManager.FEATURE_FINGERPRINT) ||
-                !packageManager.hasSystemFeature(LineageContextConstants.Features.FOD)) {
+                !FodUtils.hasFodSupport(mContext)) {
```

Comment:
FodUtils.hasFodSupport should have check for packageManager.hasSystemFeature(LineageContextConstants.Features.FOD) as some devices might be using it.

or

it can go as 

!(packageManager.hasSystemFeature(LineageContextConstants.Features.FOD) || FodUtils.hasFodSupport(mContext))

Reply:
should be best imo
```!(packageManager.hasSystemFeature(LineageContextConstants.Features.FOD) || FodUtils.hasFodSupport(mContext))```